orchestration/dagster_orchestration/hca_utils/main.py

```diff
@@ -13,6 +17,21 @@ def error(self, message):
         sys.exit(2)
 
 
+def get_api_client(host: str):
+    # get token for jade, assumes application default credentials work for specified environment
+    credentials, _ = google.auth.default()
+    auth_req = Request()
+    credentials.refresh(auth_req)
```

Comment:
Any reason you can't inline the construction of the empty `Request` object here rather than assigning to the throwaway `auth_req` var? Doesn't look like it's used anywhere  else.